go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/chromedp/chromedp v0.1.1
 	github.com/codeship/codeship-go v0.0.0-20180717142545-7793ca823354
 	github.com/denormal/go-gitignore v0.0.0-20180713143441-75ce8f3e513c
+	github.com/docker/docker v0.0.0-20171206114025-5e5fadb3c020
```

Comment:
Docker?

Reply:
Pushed by mistake, don't know how it's been upgraded :/